solidity/deploy/18_transfer_reimbursement_pool_ownership.ts

```diff
@@ -0,0 +1,19 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts, helpers } = hre
+  const { deployer, governance } = await getNamedAccounts()
+
+  await helpers.ownable.transferOwnership(
```

Comment:
Unfortunately, we can not transfer the `ReimbursementPool` ownership there because we need to authorize `ecdsa` `WalletRegistry` and `tbtc-v2` `MaintainerProxy` in the `ReimbursementPool`. If we transferred the ownership in `beacon` where the pool is deployed, we would need to ask council (or DAO) for the rest of authorizations during the deployment.